backend/app/classification/cache_helpers.py

```
@@ -0,0 +1,222 @@
+from cachetools import TTLCache
+from collections import Counter
+from langchain_core.messages import HumanMessage
+import xxhash
+import hashlib
+import json
+
+logger = logging.getLogger(__name__)
+
+try:
+    _HAS_XXHASH = True
+except Exception:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)